go.mod

```diff
@@ -9,6 +9,9 @@ replace (
 	// required by https://github.com/kubernetes/code-generator/blob/master/go.mod
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.5.5
 
+	// TEMPORARY replacement until a new helm release version gets released.
+	helm.sh/helm/v3 => github.com/antgamdia/helm/v3 v3.8.0
+
```

Comment:
Temporarily added for letting the CI effectively test the changes; will be removed as soon as Helm releases a new version